src/containers/Dashboard.vue

```diff
@@ -54,6 +54,12 @@
       @hide="showTourEndWindow = false"
       v-if="showTourEndWindow"
     />
+    <div class="dashboard-builtBy">
+      Built with ❤️ by
```

Comment:
@einc can you drop a heart svg for Jason? We should use the svg version of the emoji instead of ascii

Reply:
Also I'm not sure if the color for "Built with .. by..." is exactly right. I can't seem to find the Figma file for this... @einc can you drop the Figma link?

Reply:
interaction with bottom bar seems fine to me! and no to fixed footer